### PR TITLE
DOCS-4502 Getting Started with Feature Flag data in RUM

### PR DESCRIPTION
**Pull request by meghanlo**
### What does this PR do?

Adding a guide to set up collection of feature flag data in RUM (private beta)

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by drichards-87** [comment]
@meghanlo Thanks for submitting this new guide! I've created a docs ticket for the editorial review. It will be assigned during the team's docs planning on Monday.

File: content/en/real_user_monitoring/guide/setup-feature-flag-data-collection.md [hunk 1]
```diff
@@ -0,0 +1,406 @@
...
+Feature Flag data collection in RUM is in private beta. To request access, contact Support at support@datadoghq.com.
+{{< /beta-callout >}}
+
+## Overview
+Feature flag data gives you greater visibility into your user experience and performance monitoring by allowing you to determine which users are being shown a specific feature and if your change is having an impact on your user experience or causing a negative impact in performance. 
+
+By enriching your RUM data with feature flag data, you can be confident that your feature will successfully launch, and ensure it's not unintentionally causing a bug or performance regression. With this additional layer of insight, you can now correlate feature releases with performance, pinpoint issues to specific releases, and troubleshoot faster.
+
+## Setup
+Feature Flag tracking is available in the RUM Browser SDK. To start, set up [RUM browser monitoring][1]. You need the Browser RUM SDK version >= 4.25.0.
```
Comment (rtrieu) [inline content/en/real_user_monitoring/guide/setup-feature-flag-data-collection.md:25]:
```suggestion
Feature flag tracking is available in the RUM Browser SDK. To start, set up [RUM browser monitoring][1]. You need the Browser RUM SDK version >= 4.25.0.
```

File: content/en/real_user_monitoring/guide/setup-feature-flag-data-collection.md [hunk 2]
```diff
@@ -0,0 +1,406 @@
...
+
+Feature Flags appear in the context of your RUM Sessions, Views, and Errors as a list. 
+
+{{< img src="real_user_monitoring/guide/setup-feature-flag-data-collection/feature-flag-list-rum-event.png" alt="Feature Flag list of attributes in RUM Explorer" style="width:75%;">}}
+
+### Search Feature Flags using the RUM Explorer
+Search through all the data collected by RUM in the [RUM Explorer][7] to surface trends on feature flags, analyze patterns with greater context, or export them into [dashboards][8] and [monitors][9]. You can search your Sessions, Views, or Errors in the RUM Explorer, with the `@feature_flags.{flag_name}` attribute. 
+
+#### Sessions
+Filtering your Sessions with the `@feature_flags.{flag_name}` attribute, you will find all Sessions in the given time frame where your feature flag was evaluated.
```
Comment (rtrieu) [inline content/en/real_user_monitoring/guide/setup-feature-flag-data-collection.md:381]:
```suggestion
Filtering your **Sessions** with the `@feature_flags.{flag_name}` attribute, you will find all sessions in the given time frame where your feature flag was evaluated.
```

File: content/en/real_user_monitoring/guide/setup-feature-flag-data-collection.md [hunk 3]
```diff
@@ -0,0 +1,406 @@
...
+
+{{< img src="real_user_monitoring/guide/setup-feature-flag-data-collection/rum-explorer-session-feature-flag-search.png" alt="Search Sessions for Feature Flags in the RUM Explorer" style="width:75%;">}}
+
+#### Views
+Filtering your Views with the `@feature_flags.{flag_name}` attribute, you will find the specific Views in the given time frame where your feature flag was evaluated.
```
Comment (rtrieu) [inline content/en/real_user_monitoring/guide/setup-feature-flag-data-collection.md:386]:
```suggestion
Filtering your **Views** with the `@feature_flags.{flag_name}` attribute, you will find the specific views in the given time frame where your feature flag was evaluated.
```

File: content/en/real_user_monitoring/guide/setup-feature-flag-data-collection.md [hunk 4]
```diff
@@ -0,0 +1,406 @@
...
+
+{{< img src="real_user_monitoring/guide/setup-feature-flag-data-collection/rum-explorer-view-feature-flag-search.png" alt="Search Views for Feature Flags in the RUM Explorer" style="width:75%;">}}
+
+#### Errors
+Filtering your Errors with the `@feature_flags.{flag_name}` attribute, you will find all the Errors in the given time frame that occurred on the View where your feature flag was evaluated
```
Comment (rtrieu) [inline content/en/real_user_monitoring/guide/setup-feature-flag-data-collection.md:391]:
```suggestion
Filtering your **Errors** with the `@feature_flags.{flag_name}` attribute, you will find all the errors in the given time frame that occurred on the View where your feature flag was evaluated
```

**Review by rtrieu — CHANGES_REQUESTED** [review]
Mostly edits to make capitalizations consistent with our guidelines, and a few word cuts. Thanks for submitting this new guide!

**Review by rtrieu — APPROVED** [review]

**Pull request merged**